edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/edgedeployment/deployment/KubernetesDeploymentMapper.cs

```diff
@@ -336,8 +336,21 @@ List<V1EnvVar> CollectModuleEnv(KubernetesModule module, IModuleIdentity identit
                 .ForEach(mounts => volumeMountList.AddRange(mounts));
 
             // collect volumes and volume mounts from HostConfig.Mounts section for binds to host path
-            var bindMounts = module.Config.CreateOptions.HostConfig
-                .FlatMap(config => Option.Maybe(config.Mounts))
+            var mountListOption = module.Config.CreateOptions.HostConfig
+                .FlatMap(config => Option.Maybe(config.Mounts));
+
+            mountListOption.ForEach(mounts =>
+                {
+                    foreach (var mount in mounts)
+                    {
+                        if (mount.Type is null || mount.Source is null || mount.Target is null)
```

Comment:
well, my csharp became a bit rusty, what does this mean?
`mount.Type is null` - it looks sql to me :D

Reply:
https://devblogs.microsoft.com/dotnet/new-features-in-c-7-0/#is-expressions-with-patterns